### DISCUSSION TITLE
How does it compare to against Prologue ?

### DISCUSSION THREAD

**Discussion by imxade** [HapticX/happyx#64] [Q&A] [answered]
Newbie here, just wanted to get some thoughts.
will I be missing out something if I choose Happyx over [prologue](https://github.com/planety/prologue) ?
Don't wanna spend too much time, just on learning one.

**Comment by Ethosa** [comment]
Depends on what you're going to do

**Reply by imxade** [reply]
webApps:
data-visualization
e-commerce

Blogging with markdown
similar stuff

**Reply by Ethosa** [reply, marked as answer]
@imxade 
you can easily use HappyX for these things! 🙂

**Comment by Ethosa** [comment]
But, basically, you won't miss anything.

**Comment by Ethosa** [comment]
I opened [this issue](https://github.com/HapticX/happyx/issues/78) to write in Wiki page comparison of HappyX and other web frameworks.

If you may, please edit [HappyX For Prologue Programmers](https://github.com/HapticX/happyx/wiki/HappyX-For-Prologue-Programmers)
